packages/issm/package.py

```diff
@@ -25,6 +26,11 @@ class Issm(AutotoolsPackage):
     depends_on("petsc+metis+mumps+scalapack")
     depends_on("m1qn3")
 
+    def url_for_version(self, version):
+        url = "https://github.com/ISSMteam/ISSM/archive/refs/tags/v{0}.tar.gz".format(version)
+        return url
```

Comment:
Does our usual syntax and github URL not work? i.e.
```
        return "https://github.com/ISSMteam/ISSM/tarball/{0}".format(version)
```

Reply:
I don't think I tried that one. I did copy one from another package that was different again, and it definitely didn't work.

I have changed to the suggested format above and it works. Note that they're using `vX.X` as their tags, so I've made the change as above, but pre-pending a `v`.